homeassistant/components/sharkiq/manifest.json

```
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/sharkiq",
   "iot_class": "cloud_polling",
   "loggers": ["sharkiq"],
-  "requirements": ["sharkiq==1.4.0"]
+  "requirements": ["sharkiq==1.4.2"]
```

```suggestion
  "requirements": ["sharkiq==1.4.2", "auth0-python>=4.10.0"]
```
Sorry @Freebien one more thing caught my attention.  Previously the `auth0-python` library was not used, but the changes we made upstream make it required.  Could you apply this update to the manifest file and then re-run the requirements generation script?  That will ensure the core repo `requirements.txt` file gets updated as well.

---

I mean, if you install a dependency it should install all dependencies that are required by that integration, so if this dependency is declared in your integration it should work

---

I just wait @JeffResc's answer before sending the last update.

---

@Freebien can you test your branch in a dev container and see if the auth0 dependency is added to `requirements_all.txt`?